cmd/basin/config.go

```diff
@@ -10,7 +10,7 @@ import (
 )
 
 // DefaultProviderHost is the address of Basin Provider.
-const DefaultProviderHost = "34.73.36.206:3000"
+const DefaultProviderHost = "basin.tableland.xyz:3000"
```

Comment:
now it points to the k8s deployment

Reply:
do we need to ensure that client node that is running the basin cli is using the correct DNS server? not sure if we can do anything at our end if they are not able to resolve the domain name but maybe something to mention in doc?

Reply:
the only way to not use the correct DNS server is if the client intentionally decides to use a different one by using the `--provider` flag. in that case, there's nothing we can do if the chosen provider does not work. 

makes sense or maybe I misunderstood your question?

Reply:
by dns server i meant the server that resolves the dns. for instance, due to some misconfiguration on the host it could not resolve "basin.tableland.xyz" and hence could not connect to the provider. but i think, the error message should be enough to tell the user what is going on